ui/src/Parties.ts

```diff
@@ -1,30 +1,28 @@
-import { convertPartiesJson, PartyDetails } from '@daml/hub-react'
-
-import { ledgerId } from './config'
+import { convertPartiesJson, PartyToken } from '@daml/hub-react'
 
 const PARTIES_STORAGE_KEY = 'imported_parties';
 
-export function storeParties(parties: PartyDetails[]): void {
+export function storeParties(parties: PartyToken[]): void {
     localStorage.setItem(PARTIES_STORAGE_KEY, JSON.stringify(parties));
 }
 
-export function retrieveParties(validateParties: boolean = true): PartyDetails[] | undefined {
+export function retrieveParties(publicParty: string, validateParties: boolean = true): PartyToken[] | undefined {
```

Comment:
`retrievePartyTokens`? Just for clarity bc the object changed?

Reply:
Hmm... `PartyToken` contains more than just the token, but also all the info about the party too (that was embedded in the token)